testutils/ws_test.go

```diff
@@ -44,10 +44,10 @@ func TestWsNotify(t *testing.T) {
 		return
 	}
 	spendAmt := types.Amount{Value: 50 * types.AtomsPerCoin, Id: types.MEERID}
-	txid := Spend(t, h, spendAmt)
+	txid, addr := Spend(t, h, spendAmt)
 	t.Logf("[%v]: tx %v which spend %v has been sent", h.Node.Id(), txid, spendAmt.String())
 	t.Log(h.Wallet.Addresses())
-	AssertMempoolTxNotify(t, h, txid.String(), h.Wallet.Addresses()[1], 5)
+	AssertMempoolTxNotify(t, h, txid.String(), addr.String(), 5)
```

Comment:
should not the spend keep in  the order of addresses? 
if ```addr != wallet.addresses[1]```, does not mean some kind of double there?